frontend/src/config/translations.ts

```diff
@@ -19,6 +19,8 @@ export const translations: Translations = {
     "nav.Events": "Events",
     "nav.Announcements": "Announcements",
     "nav.ContactUs": "Contact Us",
+    "icon.Menu": "Menu",
+    "icon.Close": "Close",
```

Comment:
i think you also need to add these keys "icon.Menu" and "icon.Close" to the type `TranslationEntryKey` on line 7
i.e.
`export type TranslationEntryKey = PrefixedNavRoute | "home" | "icon.Menu" | "icon.Close" ;`

this is so that intellisense can auto-fill the keys when using the translation function `t`

Reply:
I've updated but not fully understanding this.

So why we are not adding nav. ones and adding "home", "icon.Menu", and "icon.Close"? What's the differences?

Reply:
PrefixedNavRoute covers the types for nav